src/server/requestHandler.ts

```diff
@@ -48,10 +48,13 @@ export function createRequestHandler(options: CreateRequestHandlerOptions) {
   }
 
   return async function requestHandler(request: Request): Promise<Response> {
-    const { raw, transpile } = await assets(sourceDirectory);
+    const { apiRoutes, raw, transpile } = await assets(sourceDirectory);
```

Comment:
It's a shame this have to be recreated on each request. Maybe we could memoize `assets()`